app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -43,25 +51,51 @@ window.GOVUK.analyticsGA4.analyticsModules = window.GOVUK.analyticsGA4.analytics
         return
       }
 
-      if (this.isMailToLink(href)) {
+      var linkAttributes = element.getAttribute('data-ga4-link')
+      if (linkAttributes) {
+        linkAttributes = JSON.parse(linkAttributes)
+        clickData = Object.assign(clickData, linkAttributes)
+
+        /* Since external links can't be determined in the template, we use _ as a signal
+        for our JavaScript to determine this value. */
+        if (clickData.external === '_' && clickData.url) {
```

Comment:
I'm not sure I understand why this is using `_` as a flag - why can't it use `true/false`?